src/api/catalog-search/catalog-search.ts

```diff
@@ -92,6 +92,7 @@ export interface CatalogSearchParams {
   query?: string;
   filters?: CatalogSearchFilters;
   sort?: CatalogSearchSort;
+  dedup?: boolean;
```

Comment:
I think I'd prefer this not to be optional so that behavior is consistent across the board. As is, we'll still get duplicated results from search suggestions. LMK what you think, we could also default this to true as an alternative